docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,55 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
 </runtime>
 ```
 
+## What's new in accessibility in .NET Framework 4.8.1
+.NET Framework 4.8.1 includes new accessibility features in the following areas:
+
+- [Windows Forms](#winforms481)
+
+- [Windows Presentation Foundation (WPF)](#wpf481)
+
+<a name="winforms481"></a>
+
+### Windows Forms
+**Added and improved UIA representations**
+
+Prior to .NET Framework 4.8.1 Windows Forms was missing support for a range of UIA patterns to support assistive technology like to interact with applications. This could cause Narrator or other screen readers to report incomplete or incorrect information, and prevent important functionality like moving a cursor through the text in a <xref:System.Windows.Forms.TextBox> control. With this version of .NET Framework all of the requried patterns for the Common Controls have been implemented giving users of assistive technology a much richer application interaction experience.
```

Comment:
The first sentense may need some tweaks as it looks as though it has broken grammar: "...WF was missing support for ... to support assistive technology like to interact with applications."